tests/test_todo_integration_tests.py

```diff
@@ -1,48 +1,102 @@
 
 import pytest
 import dotenv
+import os
 from requests.api import patch
 from todo_app.app import create_app as create_app
-from unittest.mock import *
+from unittest.mock import  Mock, patch 
+
+trelloBoardUrl = 'https://api.trello.com/1/boards'
 
 @pytest.fixture
 def client():
     # Use our test integration config instead of the 'real' version
     file_path = dotenv.find_dotenv('.env.test')
     dotenv.load_dotenv(file_path, override=True)
     # Create the new app.
-    test_app = create_app()
+    with patch('requests.get') as mock_get_requests:
+        mock_get_requests.side_effect = mock_get_responses
+        test_app = create_app()
+
     # Use the app to create a test_client that can be used in our tests.
     with test_app.test_client() as client:
         yield client
 
-def test_loading_page(client):
-    res = client.get('/')
-    assert res.status_code == 200
-
-
-#Haven't check if the next ones actually work - no time
 @patch('requests.get')
 def test_get_list(mock_get_requests, client):
-    mock_get_requests.side_effect = mock_get_lists("https://api.trello.com/1/boards/6050cee9111302705ff69916/lists")#ideally would read Id from the secrets file
-    response = client.get('/')           
-    assert response.status_code == 200
-      
-def mock_get_lists(url):
-    if url == f'https://api.trello.com/1/boards/6050cee9111302705ff69916/lists':
+    mock_get_requests.side_effect = mock_get_responses
+    response = client.get('/')    
+    response_text = response.get_data(as_text=True)    
+    assert 'No recent items completed yet' in response_text 
+    assert 'No items in progress' in response_text 
+
+def mock_get_responses(url):
```

Comment:
Tests currently fail because this function doesn't match how your code is trying to use requests.get.

Your app tries to run e.g.
```
requests.get(
        urlCards.format(boardId),
        params=queryCards
    )
```

When your test runs, it replaces requests.get with mock_get_responses. You then get an error when your app tries to pass a `params=...` argument to mock_get_responses. 

```suggestion
def mock_get_responses(url, params):
```